ansible/roles/cluster_infra/templates/resources.tf.j2

```diff
@@ -224,7 +257,29 @@ resource "openstack_networking_port_v2" "control" {
   }
 }
 
+# Storage VLAN
+
+resource "openstack_networking_port_v2" "control_portal_storage" {
+  name           = "{{ cluster_name }}-control-0-portal-storage"
+  network_id     = data.openstack_networking_network_v2.portal_storage_direct.id
```

Comment:
again why `portal`? `storage` I think makes some sense c.f. something more generic, implies it probably does not provide external connectivity, and hopefully high perf.